src/manifolds/Rotations.jl

```diff
@@ -307,6 +307,10 @@ Tangent vectors are represented by matrices.
 """
 inner(M::Rotations, x, w, v) = dot(w, v)
 
+function inverse_represent_in_basis(M::Rotations, x, v, B::ArbitraryOrthonormalBasis) where N
+    return hat(M, x, v)
```

Comment:
There's a missing factor of √2 to make this an ONB.

```suggestion
    T = Base.promote_eltype(x, v)
    return hat(M, x, v) ./ sqrt(T(2))
```